Northstar.Custom/mod/scripts/vscripts/sh_bleedout_damage.gnut

```diff
@@ -102,60 +131,83 @@ void function InitClientBleedoutForLocalPlayer( entity player )
 #if SERVER
 void function HandleDamageForBleedout( entity player, var damageInfo )
 {
-	if ( IsInstantDeath( damageInfo ) || DamageInfo_GetForceKill( damageInfo ) || player.IsTitan() || file.bleedingPlayers.contains( player ) )
+	// doing a better check through _bleedout.gnut
+	if ( IsInstantDeath( damageInfo ) || DamageInfo_GetForceKill( damageInfo ) || player.IsTitan() )	
 		return
+
+	// handling downed player damages
+	if( Bleedout_IsPlayerBleeding( player ) )
+	{
+		if ( player.GetHealth() + player.GetShieldHealth() - DamageInfo_GetDamage( damageInfo ) <= 0 )
+		{	
+			entity attacker = DamageInfo_GetAttacker( damageInfo )
+			entity realAttacker = Bleedout_GetLastAttacker( player )
+			if( IsValid( attacker ) && IsValid( realAttacker ) ) // real attacker exists
+			{
+				if( realAttacker.IsPlayer() && attacker.IsPlayer() && realAttacker != player && realAttacker != attacker )
+				{ // player is killed by other players, redirect score to knockdowner, can't handle executions!
+					DamageInfo_SetDamage( damageInfo, player.GetHealth() - 1 )
+					player.TakeDamage( player.GetMaxHealth() + 1, 
+									   realAttacker, 
+									   realAttacker, 
+									   { // damage table
+										weapon = Bleedout_GetLastAttackerWeapon( player ), // do this work? for supporting tactical_cdr_on_kill
+										origin = DamageInfo_GetDamagePosition( damageInfo ), 
+										force = DamageInfo_GetDamageForce( damageInfo ), 
+										scriptType = DamageInfo_GetCustomDamageType( damageInfo ), 
+										damageSourceId = eDamageSourceId.bleedout // proper damageSource for now
+									   })
```

Comment:
Is that large indentation actually our Respawn based formatting? Feels a bit off

@RoyalBlue1 to double check ^^

Reply:
Respawn sometimes formats like this.

Reply:
:dread: